pom.xml

```diff
@@ -199,6 +199,25 @@
         </exclusions>
       </dependency>
       <dependency>
+        <groupId>org.apache.jena</groupId>
+        <artifactId>jena-arq</artifactId>
+        <version>${jena.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+          </exclusion>
+          <exclusion>
```

Comment:
Yes, there is a version conflict between the version of jsonld-java that jena pulls in and the most recent jsonld-java release.